SofaGLFW/src/SofaGLFW/SofaGLFWGUI.cpp

```diff
@@ -53,7 +53,7 @@ int SofaGLFWGUI::closeGUI()
     return 0; 
 }
 
-void SofaGLFWGUI::setScene(sofa::simulation::NodeSPtr groot, const char* filename, bool temporaryFile)
+void SofaGLFWGUI::setScene(simulation::NodeSPtr groot, const char* filename, bool temporaryFile)
```

Comment:
I don't understand why you removed a `sofa` namespace.

Reply:
I removed the sofa namespace to avoid redundancy in the code, and I applied this change to all files with redundant declarations in SofaGLFW.